README.md

```diff
@@ -1,70 +1,152 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <img src="./magicians.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [ ] Good job setting up the README file for documenting your work , however it could be better to avoid broken link in your file to make it more professional 👏 , Kindly solve it

Reply:
comment removed